docs: update doc for OVMS change and benchmarking (#219)
intel-retail/automated-self-checkout: docs_src/pipelinerun.md

You can run the pipeline script, `docker-run.sh`, with the following input parameters:

1. Media type
    - Camera Simulator using RTSF
    - Intel® RealSense™ Camera
    - USB Camera
    - Video File
2. Platform
    - core
    - dgpu.0
    - dgpu.1
    - xeon
3. [Optional parameters](#optional-parameters)
 

> Run the command based on your requirement. You have to get your choices for #1, #2, #3 above to start the pipeline run, see [details](#run-pipeline-with-different-input-sourceinputsrc-types) section below.

Reviewer: ```suggestion
Run the command based on specific requirements. Select either options #1, #2 or #3 above to start the pipeline run, see [details](#run-pipeline-with-different-input-sourceinputsrc-types) section below.
```

| RealSense camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc <serial_number> --realsense_enabled</code>        |
| USB camera        | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc /dev/video0</code>                         |
| Video file      | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc file:my_video_file.mp4</code>             |

> **_Note:_**  For simulated camera as input source, please [run camera simulator first](./run_camera_simulator.md).

Reviewer: ```suggestion
**_Note:_**  For simulated camera as input source, [run camera simulator first](./run_camera_simulator.md).
```

ls -l results
```

The **results** directory contains three types of log files:

    - **pipeline#.log** files for each pipeline/workload that is running and is the pipeline/workload current FPS (throughput) results.
    - **r#.jsonl** for each of pipeline/workload that is running and is the pipeline/workload inference results.
    - **gst-launch_device_#.log** for gst-launch console output helping for debug; the `device` in file name can be core|dgpu|xeon.

> The **#** suffixed each log file name corresponding to each pipeline run index number.

Reviewer: ```suggestion
The **#** suffixed to each log file name corresponds to each pipeline run index number.
```
